lib/debug/local.rb

```diff
@@ -105,5 +109,240 @@ def after_fork_parent
       }
     end
   end
+
+  class UI_LocalTuiConsole < UI_LocalConsole
+    attr_reader :screen
+
+    def initialize(windows)
+      super()
+
+      @original_stdout = $stdout.dup
+      @original_stderr = $stderr.dup
+      @screen = Screen.new(width, height, windows, @original_stdout)
+
+      tui_console = self
+      original_stdout = @original_stdout
+      original_stderr = @original_stderr
```

Comment:
Locals are for referencing from `define_singleton_method` (can't use ivar there), and ivars are for referencing from `#deactivate`.